eatngo-core/src/main/kotlin/com/eatngo/search/schduler/SearchProductScheduler.kt

```
@@ -0,0 +1,31 @@
+)
+class SearchProductScheduler(
+    private val searchStoreRepository: SearchStoreRepository,
+    private val searchMapRedisRepository: SearchMapRedisRepository,
+) {
+    /**
+     * 상품 검색 인덱스를 업데이트합니다.
+     * 현재는 10분마다 실행되도록 설정되어 있습니다.
+     */
+    @Scheduled(fixedDelay = 10 * 60 * 1000)
+    private fun updateSearchIndex() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)